### DISCUSSION TITLE
开启了302重定向，但还是跑的服务器的流量

### DISCUSSION THREAD

**Discussion by shcat10** [AlistGo/alist#4554] [General]
用的阿里云盘OPEN，服务器是Azure的，下载和在线看视频都会产生很高的网速，检查流量消耗也确实消耗了。但是系统的磁盘好像并没有产生变化。请问怎么排查是哪里的问题。

**Comment by kingnash6** [comment]
同问我挂载了onedrive，在网页端上传文件时，发现vps没有上传流量，但有下载流量，怎么设置可以让vps不产生流量
![alist](https://github.com/alist-org/alist/assets/34468476/1beac82d-6b07-4696-9304-201ba98aaf89)

**Comment by Julian-Won** [comment]
同样遇到该问题

**Comment by NikoCat233** [comment]
上传均走的服务器，隔壁cloudreve都是直接走的onedrive的接口。。。
